agenta-cli/agenta/sdk/agenta_decorator.py

```diff
@@ -399,12 +399,32 @@ def handle_terminal_run(
     agenta.config.set(**args_config_params)
 
     loop = asyncio.get_event_loop()
+
+    # Initialize tracing
+    tracing = agenta.llm_tracing()
+
+    # Start tracing
+    tracing.start_parent_span(
+        name=func.__name__,
+        inputs=args_func_params,
+        config=args_config_params,
+        environment="shell",  # type: ignore
```

Comment:
It cloud be not shell, right?
It could be the production environment if the user just pulled the configuration from production and is using it. We need to think about that. What would the workflow look like

Reply:
Actually, I see that this PR is about getting the observability to run when using shell, and not in the general case when the user is hosting their own app, right?